clang/test/CAS/fcache-compile-job-serialized-diagnostics.c

```diff
@@ -24,16 +24,37 @@
 // RUN: c-index-test -read-diagnostics %t/diags 2>&1 | FileCheck %s --check-prefix=SERIALIZED-HIT
 
 // CACHE-HIT: remark: compile job cache hit
-// CACHE-HIT: warning: call to undeclared function
+// CACHE-HIT: warning: some warning
 
-// CACHE-MISS: warning: call to undeclared function
+// CACHE-MISS: warning: some warning
 // CACHE-MISS-NOT: remark: compile job cache hit
 
-// FIXME: serialized diagnostics should match the text diagnostics rdar://85234207
-// SERIALIZED-HIT: warning: compile job cache hit for
+// FIXME: serialized diagnostics should include the "compile job cache" remark but without storing
```

Comment:
This drops the reference to radar number which I don't like (or you can claim to fix that radar in the description?)

Also bonus point if you just write the testcase here for the correct condition but disabled for now.

Reply:
I removed the FIXME because for its comment "serialized diagnostics should match the text diagnostics" I considered it fixed. This comment fits with where it shows up (above the check line that indicates we don't get the compilation diagnostics).

But the radar seems to actually be about a different thing than what the comment says, about remapping the file paths that are shown in diagnostics from canonical paths back to what the host has.
I could re-introduce the radar number with a new comment text but it doesn't seem to fit in that particular location, or that test in general (the remapping issue is more general than serialized diagnostics).

Probably such radar comment would fit somewhere in the code itself.

Reply:
Added a FIXME comment that references this radar number in code.